app/code/Magento/Sales/Test/Unit/Model/Order/Shipment/TrackTest.php

```diff
@@ -14,9 +14,12 @@ class TrackTest extends \PHPUnit\Framework\TestCase
 
     protected function setUp()
     {
+        $objectManager = new \Magento\Framework\TestFramework\Unit\Helper\ObjectManager($this);
         $objectManagerHelper = new \Magento\Framework\TestFramework\Unit\Helper\ObjectManager($this);
+        $zendValidatorUri = $objectManager->getObject(\Zend\Validator\Uri::class);
         $arguments = [
             'shipmentRepository' => $this->createMock(\Magento\Sales\Model\Order\ShipmentRepository::class),
+            'zendValidatorUri' => $zendValidatorUri,
```

Comment:
While this test may now pass, it's best to also cover the clients of this dependency.